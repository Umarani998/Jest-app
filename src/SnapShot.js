import React from 'react';


function Snapshot(){
    return (
        <div>
            <input type="text">email</input>
            <button onClick={() =>"http://www.facebook.com"}>Facebook</button>
        </div>
    )
}
export default Snapshot