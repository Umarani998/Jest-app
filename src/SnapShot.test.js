import SnapShot from "./SnapShot";
import renderer from "react-test-renderer";

test("renders correctly",()=>{
    const tree=renderer.create(<SnapShot/>).toJSON();
    expect(tree).toMatchSnapshot()
})