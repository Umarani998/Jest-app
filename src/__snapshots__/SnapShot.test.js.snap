// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <input
    type="text"
  >
    email
  </input>
  <button
    onClick={[Function]}
  >
    Facebook
  </button>
</div>
`;
